tests/ices.zunit

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env zunit
+
+@setup {
+  ZPLUGINS=$ZINIT[PLUGINS_DIR]
+
+  function _zunit_assert_not_exists() {
+    local pathname=$1 filepath
+
+    # If filepath is relative, prepend the test directory
+    if [[ "${pathname:0:1}" != "/" ]]; then
+      filepath="$testdir/${pathname}"
+    else
+      filepath="$pathname"
+    fi
+
+    [[ ! -e "$filepath" ]] && return 0
+
+    echo "'$pathname' does exist"
+    exit 1
+  }
+
+}
+
+@test 'mv' {
```

Comment:
Do these tests verify this fix?

Reply:
As in, are these tests running interactively?

Reply:
They are running non-interactively, as all zunit tests. I'm not aware of any way to run them interactively (which probably is because of my zsh skill) :-(

Given that the bug this should fix is about making zinit able to run in (non-interactive) scripts, this is exactly what is getting fixed.

Reply:
I did run with this fix for the last weeks and redid my setup multiple times, so I'm fairly confident it will not break interactive sessions.